modules/godot/systems/timer_updater_system.cpp

```diff
@@ -0,0 +1,30 @@
+#include "timer_updater_system.h"
+#include "core/os/os.h"
+#include "scene/main/scene_tree.h"
+
+void timer_updater_system(TimersDatabag *td,
+		SceneTreeInfoDatabag *p_sti,
+		OsDatabag *p_os) {
+	ERR_FAIL_COND_MSG(td == nullptr, "The `TimersDatabag` `Databag` is not part of this world. Add it please.");
+	ERR_FAIL_COND_MSG(p_sti == nullptr, "The `SceneTreeDatabag` `Databag` is not part of this world. Add it please.");
+	ERR_FAIL_COND_MSG(p_os == nullptr, "The `OsDatabag` `Databag` is not part of this world. Add it please.");
+	ERR_FAIL_COND_MSG(p_os->get_os() == nullptr, "No OS found. This should not happen.");
```

Comment:
Initially I was also checking it, but now it's no more needed since we have Unit Tests that make sure the systems get always a valid pointer. Removing it is better to keep the System code simpler.